src/thor/timedep_forward.cc

```diff
@@ -136,57 +145,114 @@ bool TimeDepForward::ExpandForward(GraphReader& graphreader,
     // Decide if we should expand a shortcut or the non-shortcut edge...
 
     // We didn't add any shortcut of the uturn, therefore evaluate the regular uturn instead
-    disable_uturn = ExpandForwardInner(graphreader, pred, nodeinfo, pred_idx, uturn_meta, tile,
-                                       offset_time, destination, best_path) ||
-                    disable_uturn;
+    disable_uturn =
+        ExpandInner<expansion_direction>(graphreader, pred, opp_pred_edge, nodeinfo, pred_idx,
+                                         uturn_meta, tile, offset_time, destination, best_path) ||
+        disable_uturn;
   }
 
   return disable_uturn;
 }
 
+template bool TimeDepForward::Expand<TimeDepForward::ExpansionType::reverse, BDEdgeLabel>(
```

Comment:
Explicit declaration of reverse implementation so that the TimeDepReverse translation unit can find the template instantiation at link time.  As the code stands, it's a bit weird to have a derived class trying to use template methods from the parent TU - I expect to refactor this detail away as I remove more code.